File: client-python/consumerCadastroZoo.py
from suds.client import Client
from suds.cache import NoCache
url="http://localhost:8088/server-java/CadastroZooService?WSDL"
client = Client(url, cache=NoCache())
print client
print 'Lista: ', client.service.listarAnimal()
animal = {
    'id': 0,
    'nome':'Rex',
    'raca':'viralata'
}
client.service.salvar(animal)

File: client-node/consumerCadastroZoo.js
var soap = require('soap');
var url = 'http://localhost:8080/server-java/CadastroZooService?WSDL';
soap.WSDL.prototype.ignoredNamespaces = [];
var animal = {
    'id': 0,
    'nome':'Rexy',
    'raca':'pug'
}
soap.createClient(url, function(err, client) {
    client.listarAnimal(function(err,result,res){        
        console.log('Lista:',result.return);
    })
    client.salvar(animal, function(err,result,res){
         console.log('Método salvar, sem retorno:',result);
    })
});
